inspirehep/modules/editor/api.py

```diff
@@ -248,3 +247,17 @@ def get_workflow_and_schema(workflow_id):
         "workflow": workflow,
         "schema": load_schema(schema_url)
     })
+
+
+@blueprint_api.route('/validate_workflow', methods=['POST'])
+def validate_workflow():
+    workflow = request.json
+    data = workflow['metadata']
+    schema = load_schema('hep')
+
+    errors = get_validation_errors(data, schema)
+
+    if errors:
+        workflow['_extra_data']['validation_errors'] = errors
```

Comment:
Also we have to save the workflow, example can be found here https://github.com/inspirehep/inspire-next/blob/9d3368f950d11c62910509eb58757828c5e90b7e/inspirehep/modules/workflows/views.py#L606


```
workflow = workflow_object_class.get(workflow_id)
workflow['_extra_data']['validation_errors'] = errors
workflow.save()
db.session.commit()
```